modules/hdf5_handler/HDF5CF.cc

```diff
@@ -65,7 +65,8 @@ Var::Var(const Var *var)
     coord_attr_add_path = var->coord_attr_add_path;
 
     for (const auto &vattr:var->attrs) {
-        auto attr = new Attribute();
+        auto attr_unique = make_unique<Attribute>();
+        auto attr = attr_unique.release();
```

Comment:
I think as a general pattern, it is better to call release() right before the unique_ptr goes out of scope so that if any of the code between the make_unique<>() and the end of scope throws an exception. Not a factor in this code unless vattr is null, but for other cases the use is more complex and it could be the case.

Reply:
Will check the code to see if I accidentally release it not at the end of scope.

Reply:
Actually we cannot just release the unique_ptr at the end of the scope. It should be ### right before the use of the unique_ptr in the code in this scope. I will check that.

Reply:
Have questions about your suggestions. Will slack you instead of typing here.